app/views/idv/doc_auth/document_capture.html.erb

```diff
@@ -15,14 +15,12 @@
 
     <h1>
```

Comment:
Something I noticed in talking @anniehirshman-gsa and my equivalent implementation in #3990: Other screens will use `<h1 class="h3 my0">`, which appears somewhat similar on desktop displays, but on mobile displays is much smaller when using `h3` class. So without the class, there's quite a contrast of the heading of this page and other pages when using mobile.